language/German/strings.po

```diff
@@ -1710,7 +1740,7 @@ msgstr "Schieberegler"
 
 #: src/frontend/mame/ui/state.cpp:220
 msgid "[no saved states found]"
-msgstr ""
+msgstr "[keine gespeicherten Status gefunden]"
```

Comment:
"gespeicherte Zustände" would be better I think?

Reply:
...Speicherstände...

Reply:
Fixed